### PR TITLE
feat(ios): blur effect

### PR DESCRIPTION
**Pull request by qliqdev**
Hi. I've added Blur effect instead of grey color. I think it looks nice. Only for ios. 

For android tried using `WindowManager.LayoutParams.FLAG_DIM_BEHIND` but no effect. I think its can be applied only for dialogs bg.

### CODE REVIEW

**Comment by robingenz** [comment]
Thank you for your PR.
I will try to test it this weekend.

File: ios/Plugin/PrivacyScreenPlugin.swift [hunk 1]
```diff
@@ -35,9 +35,20 @@ public class PrivacyScreenPlugin: CAPPlugin {
...
         self.privacyViewController = UIViewController()
-        self.privacyViewController!.view.backgroundColor = UIColor.gray;
+        self.privacyViewController!.view.backgroundColor = .clear;
         self.privacyViewController!.modalPresentationStyle = UIModalPresentationStyle.overFullScreen
+        if !UIAccessibility.isReduceTransparencyEnabled {
+            let blurEffect = UIBlurEffect(style: .dark)
+            let blurEffectView = UIVisualEffectView(effect: blurEffect)
+            blurEffectView.frame = self.privacyViewController!.view.bounds
+            blurEffectView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+            self.privacyViewController!.view.addSubview(blurEffectView) //if you have more UIViews, use an insertSubview API to place it where needed
```
Comment (robingenz) [inline ios/Plugin/PrivacyScreenPlugin.swift:47]: Please remove this comment.

**Review by robingenz — CHANGES_REQUESTED** [review]

**Comment by robingenz** [comment]
LGTM. I'm just wondering if this should be the default or if you should be able to configure it.
I will think about it.
Thanks a lot already!

**Comment by inexuscore** [comment]
I just wanted to request this feature. It would be amazing if we could have a blur effect instead of a black screen. Could this be supported on both Android and iOS? Any ideas when this would be available? Thank you very much.

**Comment by robingenz** [comment]
I would like to provide a configuration option to select the background color/effect. 
Unfortunately, I have not much time at the moment. Suggestions are welcome.

**Comment by robingenz** [comment]
Please merge the `main` branch and resolve the merge conflicts so I can see the changes.

**Pull request closed without merging**